gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
`require` look up files in `$LOAD_PATH`, rspec set up the local lib to be part of load path so when you do `require 'earlgrey'`, it will look up your local lib first.

`require_relative` usually requires you to maintain the same folder structure so once you move your code around, you need to make those changes. besides, logically, this is to test the gem, not your local `cli.rb` file. I'd argue `require_relative` is only used inside a module when you know exactly where your files are